### DISCUSSION TITLE
✅ Is it possible to select text on the iOS application WebSSH ?

### DISCUSSION THREAD

**Discussion by dufayard** [isontheline/pro.webssh.net#387] [Q&A]
Hi, 

I'm unable to select text... Is there any way to do so ? I'm working on an iPad pro, with a magic mouse... 

Edition: in fact I found a way, possible to open a specific window to do so, but it is not ergonomic at all. 

Thanks

**Comment by isontheline** [comment]
Bonjour Jean-François 👋

Avez-vous essayé de rester appuyé avec le bouton principal de la souris afin de faire apparaître la popup "Sélectionner" ?

Merci,

**Reply by dufayard** [reply]
Bonjour, merci j'ai vu cette option. Mais hélas je trouve ça un peu compliqué si on veut juste sélectionner un bout de texte d'une sortie de la console, vite fait. Merci néanmoins pour votre temps.

**Reply by isontheline** [reply]
Oui j'ai plusieurs issues relatives à traiter :
* https://github.com/isontheline/pro.webssh.net/issues/144
* https://github.com/isontheline/pro.webssh.net/issues/323
* https://github.com/isontheline/pro.webssh.net/issues/201
